Utility/Connect-AzureVM.ps1

```diff
@@ -1,20 +1,20 @@
 <#PSScriptInfo
 
-.VERSION 1.0
+.VERSION 2.0
 
-.GUID b66368a8-dc27-481a-b4f3-dff65a6d42ee
+.GUID 803969d1-8f13-482b-8345-6b5b503dff25
```

Comment:
Are we sure we want to change the GUID? Won't this make it a seperate script from our former one on PS Gallery?